test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,20 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def get_metadata_max_size(cache_size, cache_line_size, number_of_cores=4096):
```

Comment:
I assume You expect `cache_size` and `cache_line_size` to be objects of type Size. Give us a hint.